deeplearning4j-nearestneighbors-parent/deeplearning4j-nearestneighbors-model/src/main/java/org/deeplearning4j/nearestneighbor/model/NearestNeighborsResult.java

```diff
@@ -12,5 +11,5 @@
 @NoArgsConstructor
 public class NearestNeighborsResult {
```

Comment:
I imagine the same question can be asked with regard to `index` as well. Truth be told: I'm just patching for `distance` here, in line with the way I found it. I guess a discussion can be had about refactoring it, but I think that would include both `index` as well as `distance` (e.g. the whole class). For now though, I think this fits into / doesn't alter the original design & "spirit" of this object.